adyen.go

```diff
@@ -71,8 +77,8 @@ func (a *Adyen) ClientURL() string {
 }
 
 // adyenURL returns Adyen backend URL
-func (a *Adyen) adyenURL(requestType string) string {
-	return a.Credentials.Env.BaseURL(APIVersion) + "/" + requestType
+func (a *Adyen) adyenURL(service string, requestType string) string {
+	return a.Credentials.Env.BaseURL(service, APIVersion) + "/" + requestType + "/" + service + "/"
```

Comment:
Seems like the issue with double `service` parameter doesn't solve. Can you please check?